cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonConstants.java

```diff
@@ -71,4 +71,32 @@ private CommonConstants() {
     public static final String INTERNAL_CONCATENATOR = "=";
     public static final String CONCATENATOR = "x0000";
     
+    // Header for API uses
+    public static final String INSTANCE_HEADER = "#####\n" 
```

Comment:
From the copyright, this header is common to all the configuration files (which must have the header). I'd suggest to call this constant "CYGNUS_IPR_HEADER" (IPR stands for "Intelectural PRoperty)" and avoid the "configuration file for apache-flume" part.